src/core/messages.ts

```diff
@@ -305,23 +448,44 @@ export class DefaultMessages
    * @param clientId The client ID of the user.
    * @param logger An instance of the Logger.
    */
-  constructor(roomId: string, channelManager: ChannelManager, chatApi: ChatApi, clientId: string, logger: Logger) {
+  constructor(
+    roomId: string,
+    options: MessageOptions | undefined,
```

Comment:
Should we make this always defined and set a default?

Reply:
Done.